airbyte_cdk/sources/declarative/transformations/dpath_flatten_fields.py

```diff
@@ -8,6 +8,24 @@
 from airbyte_cdk.sources.types import Config, StreamSlice, StreamState
 
 
+@dataclass
+class KeyTransformation:
+    config: Config
+    parameters: InitVar[Mapping[str, Any]]
+    prefix: Union[InterpolatedString, str, None] = None
```

Comment:
Nit, but for `prefix` and `suffix`, since the YAML schema is defined to only accept strings, we know that this will only ever be a string or None, so we should be able to use `Optional[str]`. If mypy doesn't complain lets switch it to that.